java/src/main/java/com/logicalclocks/hsfs/engine/CodeEngine.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021 Logical Clocks AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package com.logicalclocks.hsfs.engine;
+
+import com.logicalclocks.hsfs.EntityEndpointType;
+import com.logicalclocks.hsfs.FeatureStoreException;
+import com.logicalclocks.hsfs.TrainingDataset;
+import com.logicalclocks.hsfs.metadata.Code;
+import com.logicalclocks.hsfs.metadata.CodeApi;
+import com.logicalclocks.hsfs.metadata.FeatureGroupBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+public class CodeEngine {
+
+  private CodeApi codeApi;
+  private static String kernelEnv = "HOPSWORKS_KERNEL_ID";
+  private static String webProxyEnv = "APPLICATION_WEB_PROXY_BASE";
```

Comment:
Java convention dictates that static variables should be named with all capital letters. Also add final (`private static final`)